src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -1095,6 +1108,13 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_VIEW IMPLEMENTATION.
     ls_col_spec-allow_order_by = abap_true.
     APPEND ls_col_spec TO lt_col_spec.
 
+    IF mo_repo->has_remote_source( ) = abap_true.
```

Comment:
same

Reply:
Hi, 
commited new version for local ones too.